specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.0/FormRecognizer.json

```diff
@@ -0,0 +1,1524 @@
+        "operationId": "TrainCustomModel",
+        "consumes": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "trainRequest",
+            "in": "body",
+            "description": "Input for training custom model.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/TrainRequest"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Request to train custom model is successfully queued.",
+            "headers": {
+              "Location": {
+                "type": "string",
+                "description": "Location and ID of model being trained. Trained model status is specified in Status field in result."
+              }
+            }
+          },
+          "default": {
+            "description": "Response entity accompanying non-successful responses containing additional details about the error.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Train new model example": {
+            "$ref": "./examples/TrainBatch.json"
+          },
+          "Train new model example, subfolder filter options": {
+            "$ref": "./examples/TrainBatchWithSubFolders.json"
+          }
+        }
+      },
+      "get": {
+        "summary": "List Custom Models",
+        "description": "Get information about all trained custom models",
+        "operationId": "GetCustomModels",
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "op",
+            "in": "query",
+            "description": "Summary or full list of models to be returned.",
+            "required": false,
+            "type": "string",
+            "enum": [
+              "full",
+              "summary"
+            ]
+          }          
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/Models"
+            }
+          },
+          "default": {
+            "description": "Response entity accompanying non-successful responses containing additional details about the error.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
```

Comment:
What is "nextLink"?

Reply:
for pagination, https://github.com/microsoft/api-guidelines/blob/vNext/Guidelines.md#94-big-collections